CMakeLists.txt

```diff
@@ -118,6 +120,7 @@ add_subdirectory(${RADIUM_SRC_DIR})
 # ----------------------------------------------------------------------------------------------------------------------
 #   Applications
 #   Set Base directories for dependencies
+set( OMIT_MAIN_APP False CACHE BOOL "Choose to build or not the main radium application" )
```

Comment:
ok, so we keep it.
I believe that it should be extended to something allow to build only the lib, or the libs + apps, libs+examples, etc..

I suggest to change this variable to `BUILD_APPS̀`, and update behavior accordingly. 
Good ?